wagtailio/static/sass/base/_typography.scss

```diff
@@ -24,8 +24,10 @@ p {
 
 h1,
 .heading-one {
-    font-size: 2.8125rem;
-    line-height: 3.375rem;
+    font-size: 5.625rem;
+    line-height: 5.625rem;
+    font-weight: $weight--bold;
```

Comment:
This is set here: https://github.com/wagtail/wagtail.org/blob/next/wagtailio/static/sass/base/_typography.scss#L18

Reply:
Good spot.